chartfx-dataset/src/main/java/de/gsi/dataset/spi/DefaultAxisDescription.java

```diff
@@ -126,7 +131,8 @@ public boolean equals(final Object obj) {
      * @param epsilon allow for fuzzy comparisons of min/max
      * @return true: matches, false otherwise
      */
-    public boolean equals(final Object obj, final double epsilon) {
+    public boolean
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'equals(Object, double)' has an NPath complexity of 10368](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4730376)